cmd/juju/application/deploy.go

```diff
@@ -364,16 +364,11 @@ If a revision is specified, a channel must also be specified for Charmhub charms
 and bundles.  The charm will be deployed with revision.  The channel will be used
 when refreshing the application in the future.
 
-A local charm may be deployed by giving the path to its directory:
-
-  juju deploy /path/to/charm
-  juju deploy /path/to/charm --base ubuntu@22.04
-
```

Comment:
We should elaborate on what the local charm is to be clear. `charmcraft pack` uses the term `charm operator package`?

```
A local charm may be deploy by giving the path to the charm package:

   juju deploy /path/to/example.charm

A charm package is created with charmcraft pack command, or by creating a zip file.
```

Reply:
Agreed, but I'm dropping `, or by creating a zip file.` to illustrate that `charmcraft pack` is the recommended way. We could add developer documentation to state what makes a charm file. The help docs isn't the place for that.